setup.py

```diff
@@ -13,13 +13,19 @@
     # For pip installations
     version = str(
         ast.literal_eval(
-            open(os.path.join(
-                os.path.dirname(__file__),
-                "sdc", "crypto", "__init__.py"),
-                'r').read().split("=")[-1].strip()
+            open(
+                os.path.join(os.path.dirname(__file__), "sdc", "crypto", "__init__.py"),
+                "r",
+            )
+            .read()
+            .split("=")[-1]
+            .strip()
         )
     )
 
+with open("./README.md") as fp:
+    description = fp.read()
```

Comment:
This does seem to work but I'm not sure how relevant the info is in the Pypi package long description.

Reply:
In lieu of another long description and the fact it needs a value to suppress the warning it'll probably do. There are other examples in Pypi where the README is simply used here.